commands/psql.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+
+const co = require('co')
+const cli = require('heroku-cli-util')
+
+function * run (context, heroku) {
+  const fetcher = require('../lib/fetcher')(heroku)
+  const psql = require('../lib/psql')
+
+  const {app, args, flags} = context
+
+  let db = yield fetcher.database(app, args.database)
+  cli.console.error(`--> Connecting to ${cli.color.addon(db.addon.name)}`)
```

Comment:
@dickeyxxx can I change this to the config var, it used to be this way, and in the case of implicit attachment, there is no addon to speak of.  The use case we are trying to support is someone `config:set` the DATABASE_URL and running `heroku pg:psql` and expecting it to connect.